R/probability_estimators.R

```diff
@@ -429,10 +439,12 @@ johnson_method <- function(x,
 
   data <- reliability_data(x = x, status = status, id = id)
 
-  johnson_method_(data)
+  johnson_method_(data, method)
 }
 
-johnson_method_ <- function(data) {
+
+
+johnson_method_ <- function(data, method = "benard") {
```

Comment:
I would think that underscore functions should not have a default argument since they are called inside other functions which provide defaults.

Reply:
I kind of agree. The default argument is currently needed for the tests, as they don't set it.

I think our underscore functions can have default arguments, but then they should be consistent, that means in this example `method = c("benard", "invbeta")` and then `match.arg`. Or no default argument at all. However, a default argument never hurts.